### PR TITLE
💚 Fix CI: PDF Generation Error

### LINKED ISSUE #97: 💔 GitHub Action Workflow fails to generate a PDF
## Issue

When we click `Resources → PDF` to open the [PDF version of TrioDocs](https://docs.diy-trio.org/trio-docs.pdf), we get a 404 error because the PDF file does not exist.
It has not been created.

A [GitHub workflow](https://github.com/nightscout/trio-docs/blob/b2fdbd670925f4df14ef596b8eeeb3fde145e9a7/.github/workflows/publish.yml#L60) fails to generate the PDF fails because it cannot load a font due to a CORS error.

## Description

We use the `mkdocs-exporter` plugin to orchestrate the creation of the PDF.
Under the hood, it uses a `Playwright` Chromium headless browser to scour the website and compile the PDF.

❌ When processing a page with [MathJax notation](https://squidfunk.github.io/mkdocs-material/reference/math/?h=mathjax#mathjax), the headless Web browser tries to load a Web font and refuses to do so because of a cross-domain request and we get a [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) error:
  > Access to font at 'https://unpkg.com/mathjax@3.2.2/es5/output/chtml/fonts/woff-v2/MathJax_Calligraphic-Bold.woff' (redirected from 'https://unpkg.com/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Calligraphic-Bold.woff') from origin 'file://' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
  
To load the MathJax Javascript file, we use the MathJax URL [mentioned in the mkdocs-material documentation](https://squidfunk.github.io/mkdocs-material/reference/math/?h=mathjax#mathjax):
>  https://unpkg.com/mathjax@3/es5/tex-mml-chtml.js
  
This generic v3 URL redirects to the current version (3.2.2) which is a different URL:

> https://unpkg.com/mathjax@3.2.2/es5/tex-mml-chtml.js

I don't think the redirection is posing a problem here.
This Javascript file is large (~1.2MB) because it is not minified.

## Solution

I decided to use the minified version of the same file **but** hosted **on Cloudflare** instead of _Unpkg_:
> https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.2.2/es5/tex-mml-chtml.min.js
✅ This fixed the issue.  

## Cause

My guess is that is due to _Cloudflare_ using more permissive CORS server headers telling the server to allow the serving of fonts across domains, in our case from `file://`. 

```
Access-Control-Allow-Origin "*"
```

### PR DESCRIPTION
**Pull request by ebouchut**
TLDR; This PR fixes the build script (`deploy.yml`) which now creates the PDF.
> A remote font could not be loaded because it was on a different domain (CORS error).

📚 See Issue #97 for a detailed explanation of the error, cause, and solution.

Fixes #97

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -65,8 +65,7 @@ theme:
 
 plugins:
   - open-in-new-tab
-  - macros:
-      include_dir: .
+  - macros
```
Comment (ebouchut) [inline mkdocs.yml:68]: Removed because it is redundant.
The default is to include the current folder `.`.

**Review by ebouchut — COMMENTED** [review]
LGTM

**Comment by tmhastings** [comment]
@ebouchut do we still need this fix?

**Comment by ebouchut** [comment]
I will double check but I think these commits have been replayed on what has became the 0.4-docs branch. If so, I will close this PR.

**Comment by ebouchut** [comment]
✅ Closing this PR.
Its commits are already present in `dev`.
